gnat2goto/ireps/irep_specs/c_enum_member.json

```diff
@@ -2,6 +2,6 @@
   "namedSub": {
     "value": {"type": "string"},
     "identifier": {"type": "string"},
-    "base_name": {"type": "string"}
+    "baseName": {"type": "string"}
```

Comment:
I appreciate that this is probably needed but I wonder why this was changed in CPROVER as minor cosmetic changes to machine <-> machine formats seem a bit daft.

Reply:
AHHHHHH.... makes sense....